<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="StartDateLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>StartDate:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="StartDateTime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>5</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>5</month>
     <day>31</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="EndDateLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EndDate:</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="EndDateTime">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>6</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="AnisLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ANIS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AnisImput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DnisLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DNIS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DnisImput">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ImportLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Import:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>691</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BuscarButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
